### DISCUSSION TITLE
List with selectable elements

### DISCUSSION THREAD

**Discussion by ottosson** [mantinedev/mantine#1822] [Feature requests]
Would a regular old list with selectable elements be an interesting addition to Mantine?
I mean a list that's selectable just as a regular html-select is with the multi option enabled (shift/ctrl+click behavior, up/down keyboard arrow selection and so on), but with custom elements, unlike the html select which is limited to only have text options, not custom html elements.

**Comment by rtivital** [comment]
We already provide MultiSelect component to handle selection of multiple items, you can also build your own component with Checkbox

**Reply by ottosson** [reply]
Hm ok. The MultiSelect component is a dropdown. Is it possible to make it a regular list? Same behavior as the HTML Select element (with multi enabled)?

**Reply by rtivital** [reply]
Nope, it is only the input
